### PR TITLE
Browser compatability

### PR DESCRIPTION
**Pull request by hoptond**
Sound/images now work correctly across all browsers, especially the much loved and well engineered powerhouse of browsers that is Internet Explorer. The pertinent changes are in audio.js

### CODE REVIEW

File: test.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (patrickkenyon) [inline test.html]: Do you need this file?

**Review by patrickkenyon — COMMENTED** [review]
Just one thing...

File: test.html (hunk 1 above)
Comment (hoptond) [reply test.html]: fixed in ed3d7fb

**Review by patrickkenyon — APPROVED** [review]

File: js/audio.js [hunk 2]
```diff
@@ -36,11 +40,12 @@ function listenCityClick() {
...
-        cheer1: 'audio/cheer1.wav',
-        cheer2: 'audio/cheer2.wav',
-        cheer3: 'audio/cheer3.wav',
-        cheer4: 'audio/cheer4.wav'
+        cheer1: 'audio/cheer1.mp3',
+        cheer2: 'audio/cheer2.mp3',
+        cheer3: 'audio/cheer3.mp3',
+        cheer4: 'audio/cheer4.mp3'
     }
+    console.log(audioObj[noise])
```
Comment (mporam) [inline js/audio.js]: remove debug code

**Review by mporam — CHANGES_REQUESTED** [review]

File: js/audio.js (hunk 2 above)
Comment (hoptond) [reply js/audio.js]: fixed in 5d736a2

**Review by mporam — APPROVED** [review]

**Pull request merged**